home/forms.py

```diff
@@ -5,6 +5,12 @@
 from home.models import Point, WgUpdate, Minutes
 from users.models import Profile
 
+def validate_uoe_mail(value):
+    if "ac.uk" in value:
```

Comment:
More of an edge case, but `if value.endswith('ac.uk')` might be better in case some one with an email similar to `mac.ukulele@gmail.com` will be able to sign up.

In case you change this just a heads up `if value.endswith('ac.uk')` will crash if value is `None` but I assume that can't happen?